include/ev_cherry_manager.h

```diff
@@ -10,6 +10,12 @@ extern "C" {
 
 extern ACTOR_PROFILE Ev_Cherry_Manager_Profile;
 
+typedef struct cherry_manager_s CHERRY_MANAGER_EVENT_ACTOR;
```

Comment:
If there's no need to type it like this, then we usually just typedef it within the declaration